### PR TITLE
ubuntu-18.04: add more packages from distro

### PR DESCRIPTION
**Pull request by alkino**

### CODE REVIEW

File: sysconfig/ubuntu-18.04/packages.yaml [hunk 1]
```diff
@@ -95,6 +111,10 @@ packages:
             python@2.7.15: /usr
             python@3.6.7: /usr
         version: [3.6.7, 2.7.15]
+    py-numpy:
+        paths:
+            py-numpy@1.17.4: /usr
+        version: [1.17.4]
```
Comment (matz-e) [inline sysconfig/ubuntu-18.04/packages.yaml]: no

Comment (alkino) [reply sysconfig/ubuntu-18.04/packages.yaml]: oups sorry! I didn't want!

**Comment by alkino** [comment]
Sorry for the force push

**Comment by alkino** [comment]
Please retest

**Comment by alkino** [comment]
retest this please Jenkins

**Comment by alkino** [comment]
retest this please Jenkins

**Comment by alkino** [comment]
retest this please Jenkins

**Comment by alkino** [comment]
retest this please Jenkins

**Review by matz-e — APPROVED** [review]
I'm not entirely sure about TK, but I think the stakes here are low.

**Pull request merged**